stable/external-dns/values.yaml

```diff
@@ -81,10 +81,27 @@ azure:
   ## When a secret to load azure.json is not specified,
   ## the host's /etc/kubernetes/azure.json will be used
   ##
+  ## Deprecated: please use tenantId, subscriptionId, aadClientId and aadClientSecret values instead.
+  ##
   secretName: ""
   ## Azure resource group to use
   ##
   resourceGroup: ""
+  ## Azure tenant ID to use
+  ##
+  tenantId: ""
+  ## Azure subscription ID to use
+  ##
+  subscriptionId: ""
+  ## Azure Application Client ID to use
+  ##
+  aadClientId: ""
+  ## Azure Application Client Secret to use
+  ##
+  aadClientSecret: ""
+  ## If you use Azure MSI, this should be set to true
+  ##
+  useManagedIdentityExtension: false
```

Comment:
Please also update the **values-production.yaml** with the same changes.

Reply:
Right, fixed.